control-plane-operator/controllers/hostedcontrolplane/cno/clusternetworkoperator_test.go

```diff
@@ -53,6 +76,14 @@ func TestReconcileDeployment(t *testing.T) {
 					strconv.FormatBool(tc.expectProxyAPIServerAddress),
 					strconv.FormatBool(hasProxyAPIServerAddress))
 			}
+
+			// Verify the existing resources were preserved
+			if dep.Spec.Template.Spec.Containers[0].Resources.Requests.Cpu().MilliValue() != 100 ||
```

Comment:
Try breaking your test and see what the error output looks like. Is it useful to developers and quick to determine? Try using the golden file approach and/or `cmp.Diff()` and compare those outputs - the latter two options provide clear, minimal diffs that make it easy to a) provide test input fixures, b) declare expected output, and c) clarify what errored, how, and where.

Reply:
More succinctly - if I were to break this, I would not want to know that "some or all..." - a good test explicitly tells me which I broke, and makes it quick to determine where that failure is and how I broke it.